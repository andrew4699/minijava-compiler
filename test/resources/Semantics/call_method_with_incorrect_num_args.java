class foo {

  public static void main(String[] args) {
    System.out.println(new bar().buzz());
  }
}

class bar {
  public int buzz() {
    return this.threeArgs(1, 2);
  }

  public int threeArgs(int i, int j, int k) {
    return i + j + k;
  }
}