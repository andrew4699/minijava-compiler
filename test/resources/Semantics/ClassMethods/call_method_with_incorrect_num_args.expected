[ClassMethodsVisitor - 10]: threeArgs(int, int) does not belong to class bar
